src/components/Proposal/View/Budget/RequestedBudgetCard.css

```diff
@@ -0,0 +1,20 @@
+.RequestedBudgetCard {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 8px;
+  margin-bottom: 2px;
+}
+
+.RequestedBudgetBar {
```

Comment:
I'd move this to the DistributionBar css file

Reply:
This are classes particular to how the distribution bar looks on this `RequestedBudgetCard`. 
If some other component that uses the distribution bar, it should declare different styles (there are no defaults yet)